erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -305,12 +298,22 @@ def update_value_in_dict(data, key, gle):
 			update_value_in_dict(totals, 'total', gle)
 			if filters.get("group_by") != _('Group by Voucher (Consolidated)'):
 				gle_map[gle.get(group_by)].entries.append(gle)
+			elif filters.get("group_by") == _('Group by Voucher (Consolidated)'):
+				key = (gle.get("posting_date"), gle.get("voucher_type"),
```

Comment:
why posting_date should be part of the key?

Reply:
Done